features/manage-position/Create.tsx

```diff
@@ -71,29 +71,35 @@ const Create = () => {
   const [success, setSuccess] = useState<boolean | null>(null);
   const [error, setError] = useState<Error | null>(null);
 
+  const collateralNum = Number(collateral) || 0;
+  const tokensNum = Number(tokens) || 0;
+
   const {
     collateralRequirement: collReq,
     minSponsorTokens,
     priceIdentifier,
   } = empState;
   const liquidationPriceWarningThreshold = 0.1;
 
-  useEffect(() => {
-    if (gcr !== null) {
-      setBackingCollateralToMin(gcr, Number(tokens));
-    }
-  }, [gcr, tokens]);
-
   const setBackingCollateralToMin = (_gcr: number, _tokens: number) => {
-    // By default set amount of collateral to the minimum required by the GCR constraint. This
-    // default is intended to encourage users to maximize their capital efficiency.
+    // Set amount of collateral to the minimum required by the GCR constraint. This
+    // is intended to encourage users to maximize their capital efficiency.
     const minBackingCollateral = _gcr * _tokens;
     // We want to round down the number for better UI display, but we don't actually want the collateral
     // amount to round down since we want the minimum amount of collateral to pass the GCR constraint. So,
     // we'll add a tiny amount of collateral to avoid accidentally rounding too low.
     setCollateral((minBackingCollateral + 0.00005).toFixed(4));
```

Comment:
Didn't realize this was the solution that came out of the last PR -- very clever.